### ISSUE TITLE
Initial Launch: Category Pages

### ISSUE THREAD

**Issue by beafialho** [WordPress/wporg-news-2021#148]
The following category pages are an essential part of the redesign, therefore having them built is critical for the initial launch:

- **Podcast**

| Test Site      | Mockup |
| ----------- | ----------- |
|  <img width="1532" alt="Captura de ecrã 2022-01-05, às 12 04 58" src="https://user-images.githubusercontent.com/65220155/148214899-e2ea686c-e7f3-4f6b-bbcb-8f71147699f8.png">  |  <img width="767" alt="Captura de ecrã 2022-01-05, às 12 05 39" src="https://user-images.githubusercontent.com/65220155/148214964-240fd22f-475e-4b50-b646-e4d560ac5669.png">  |

- **Community**

| Test Site      | Mockup |
| ----------- | ----------- |
|  <img width="1534" alt="Captura de ecrã 2022-01-05, às 12 03 04" src="https://user-images.githubusercontent.com/65220155/148214654-7ecd1146-2d81-4ac3-8690-4461420099fb.png">  |   <img width="670" alt="Captura de ecrã 2022-01-05, às 12 02 39" src="https://user-images.githubusercontent.com/65220155/148214692-ff748128-4103-49ed-80d4-abc3daaae3e7.png">  |

- **Month in WP**

| Test Site      | Mockup |
| ----------- | ----------- |
|  <img width="1479" alt="Captura de ecrã 2022-01-05, às 12 07 29" src="https://user-images.githubusercontent.com/65220155/148215339-43ca496d-82b5-4246-9b77-bff26ae9e4b6.png"> |  <img width="1068" alt="Captura de ecrã 2022-01-05, às 12 08 27" src="https://user-images.githubusercontent.com/65220155/148215352-d2184e50-270a-43fa-a913-730120ce7198.png"> |

- **Releases**

| Test Site      | Mockup |
| ----------- | ----------- |
|  <img width="1527" alt="Captura de ecrã 2022-01-05, às 12 09 58" src="https://user-images.githubusercontent.com/65220155/148215491-5f0715cd-622f-4800-83b9-03bb8c5d2218.png">  |   <img width="971" alt="Captura de ecrã 2022-01-05, às 12 10 31" src="https://user-images.githubusercontent.com/65220155/148215565-6129462c-0a4d-4a88-9a1d-42023593fde7.png"> |

- **Events**

| Test Site      | Mockup |
| ----------- | ----------- |
|  <img width="1476" alt="Captura de ecrã 2022-01-05, às 12 11 40" src="https://user-images.githubusercontent.com/65220155/148215721-927060cc-20c1-4143-b9bd-6a0a8688937a.png">  |   <img width="755" alt="Captura de ecrã 2022-01-05, às 12 11 19" src="https://user-images.githubusercontent.com/65220155/148215699-5c54b808-dc0a-406f-8d7d-121587b685a6.png"> |

Let's please pay special attention to mobile as well. All mockups can be found [here](https://www.figma.com/file/O9I8Jl7kaUMTX39LlRGbGO/News-Page?node-id=4904%3A29783).

cc: @kellychoffman

**Comment by iandunn** [comment]
* **Podcast**: That's tracked in https://github.com/WordPress/wporg-news-2021/issues/130. I don't think anyone has started on it yet, the screenshot above looks like the old site rather than the test site.
* **Community**: That's tracked in https://github.com/WordPress/wporg-news-2021/issues/76. So far it's only had the some partial work on the markup and Gutenberg issues, no one has added styles yet.
* **Month in WP**: Similar to Community, this hasn't gotten styles yet
* **Events, Releases**: The sidebar content is tracked in #75, and blocked by Gutenberg. The heading size issue is a regression that's tracked in https://github.com/WordPress/wporg-news-2021/issues/145

I'm not sure there'll be time to complete Podcast/Community/Month in WP before 5.9, but it's a good stretch goal depending on how many folks pitch in. We definitely wouldn't launch them in the current state. The plan was to disable them so that they'd fall back to the default/uncategorized template.

**Comment by ryelle** [comment]
Is there anything in this issue that isn't done or tracked in another issue?

**Comment by kellychoffman** [comment]
@beafialho could you take a look?

**Comment by beafialho** [comment]
> Is there anything in this issue that isn't done or tracked in another issue?

I believe all in this issue is done or being tracked in other issues.
